api-reference/50 Common/Object Structures/dxDiagramCustomCommand/location.md

```diff
@@ -6,7 +6,11 @@ default: 'before'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a location for the command or separator on the main toolbar.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+[note]
+This property does not allow you to set the command's or separator's location in the context menu, history and view toolbars.
```

Comment:
Is "history" also a toolbar? If so, then "history and view toolbars" should be "and history and view toolbars":

```suggestion
This property does not allow you to set the location of the command or separator in the context menu, and history and view toolbars.
```